src/python/T0/RunConfig/RunConfigAPI.py

```diff
@@ -689,11 +693,13 @@ def configureRunStream(tier0Config, run, stream, specDirectory, dqmUploadProxy):
             factory = RepackWorkloadFactory()
             wmSpec = factory.factoryWorkloadConstruction(workflowName, specArguments)
             for subscription in subscriptions:
+                logging.info("subscrition_bulk %s", subscription)
```

Comment:
Typo

Reply:
This was only for debugging purposes. I updated this PR